src/colors.js

```diff
@@ -1,7 +1,12 @@
 export const colors = {
+  mainGray: '#373737',
+  dialodsGray:
+    'linear-gradient(0deg, rgba(255, 255, 255, 0.12), rgba(255, 255, 255, 0.12)), #121212;',
   lightGray: '#888',
   darkGray: '#333',
   darkViolet: '#2B2630',
+  violet: '#BB86FC',
+  turquoise: '#88EDE7',
   white87: 'rgba(255, 255, 255, 0.87)',
```

Comment:
@TomaTomoe Material design system has colors like Blue50, Blue100, Blue200, ... Blue900, BlueA100, BlueA200, BlueA400, BlueA700. Why do we use `white87` ? :)

![image](https://user-images.githubusercontent.com/6686679/95430322-4edde580-0954-11eb-8bdf-b68a833f2847.png)

https://material.io/design/color/the-color-system.html#tools-for-picking-colors

And Material UI follows the same naming:

![image](https://user-images.githubusercontent.com/6686679/95430498-80ef4780-0954-11eb-960f-5aad19fa60dd.png)

https://material-ui.com/customization/color/#examples

Reply:
And I believe it should be not `white`, but `gray`

![image](https://user-images.githubusercontent.com/6686679/95430601-a8deab00-0954-11eb-90e3-48924f107cde.png)

https://material.io/design/color/the-color-system.html#tools-for-picking-colors

Reply:
Since we use the concept already developed by Material Design as the basis for our design, we also use the styles proposed by them, possibly with several customizations. I did not see the need to additionally transfer it to our figma file. But the further, the more I understand that it was worth giving the design in a .png or .psd file